src/nn/wasm/src/bind/src/binding.cpp

```diff
@@ -18,12 +21,22 @@ using namespace tflite;
 
 namespace binding_utils {
   static gemmlowp::GemmContext gemm_context;
+  static CpuBackendContext cpu_backend_context;
+  static CpuFlags cpu_flags;
   
   // help functions
   void set_gemm_context_threads_num(int threads_num) {
     gemm_context.set_max_num_threads(threads_num);
   }
 
+  void set_cpu_context_threads_num(int max_num_threads) {
```

Comment:
Do we need to expose this function? If it is, please follow the usage of `set_gemm_context_threads_num`.

Reply:
I think we need to expose this function and I will investigate its usage in PrepareModel.js